neuromation/cli/job.py

```diff
@@ -1127,3 +1167,47 @@ def calc_statuses(status: Sequence[str], all: bool) -> Set[JobStatus]:
                 return parse_columns(format_str)
         return COLUMNS
     return format
+
+
+async def calc_life_span(client: Client, value: Optional[str]) -> Optional[float]:
+    delta = (
+        _parse_timedelta(value)
+        if value is not None
+        else await calc_default_life_span(client)
+    )
+    seconds = delta.total_seconds()
+    if seconds == 0:
+        return None
+    assert seconds > 0, f"life-span cannot be negative, got: {seconds}"
+    return seconds
+
+
+async def calc_default_life_span(client: Client) -> timedelta:
```

Comment:
Inline the function in `calc_life_span()` please.

Reply:
Why? it's harder to test

Reply:
The function is never used alone, it is always called from `calc_life_span()`.
You can test `calc_life_span(client, None)` instead of `calc_default_life_span(client)`, should prodice the same result.

Reply:
the matter of taste. changed.